dkist/io/asdf/resources/schemas/tiled_dataset-1.0.0.yaml

```diff
@@ -16,7 +16,9 @@ properties:
     items:
       type: array
       items:
-        tag: "asdf://dkist.nso.edu/tags/dataset-1.0.0"
+        anyOf:
+          - tag: "asdf://dkist.nso.edu/tags/dataset-1.0.0"
+          - tag: "asdf://dkist.nso.edu/tags/dataset-1.1.0"
```

Comment:
This might require a new version of this schema in the manifest alongside dataset-1.1.0. It should be tested to make sure old and new files work here. (This is probably fine, but it could be a pain point)

Reply:
Why would that be the case?